db/seeds.rb

```diff
@@ -1,7 +1,175 @@
+    city: "New York",
+    zip_code: "10002",
+    country_code: "US",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 6",
+    address_line1: "7316 Marshall Court",
+    city: "New York",
+    zip_code: "10002",
+    country_code: "US",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 7",
+    address_line1: "42 West Lane",
+    city: "Buffalo",
+    zip_code: "14215",
+    country_code: "US",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 8",
+    address_line1: "7216 Cottage Street",
+    city: "Brooklyn",
+    zip_code: "11208",
+    country_code: "US",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 9",
+    address_line1: "30 Atlantic Road",
+    city: "Brooklyn",
+    zip_code: "11211",
+    country_code: "US",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 10",
+    address_line1: "49 Gartner Street",
+    city: "Bronx",
+    zip_code: "10457",
+    country_code: "US",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 11",
+    address_line1: "163 Sage Ave.",
+    city: "Bay Shore",
+    zip_code: "11706",
+    country_code: "US",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 12",
+    address_line1: "9520 The Avenue",
+    city: "Harrow",
+    zip_code: "HA65 4AJ",
+    country_code: "GB",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 13",
+    address_line1: "76 Church Road",
+    city: "Canterbury",
+    zip_code: "CT8 7CI",
+    country_code: "GB",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 14",
+    address_line1: "452 Mill Lane",
+    city: "Bath",
+    zip_code: "BA16 5TJ",
+    country_code: "GB",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 15",
+    address_line1: "82 High Street",
+    city: "Edinburgh",
+    zip_code: "EH95 0CH",
+    country_code: "GB",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 16",
+    address_line1: "444 Grange Road",
+    city: "Shrewsbury",
+    zip_code: "SY47 8MF",
+    country_code: "GB",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 17",
+    address_line1: "73 Main Street",
+    city: "St Albans",
+    zip_code: "AL30 2UK",
+    country_code: "GB",
+    description: "This is a fun little store!"
+  },
+  {
+    map: maps.first,
+    name: "Location 18",
+    address_line1: "426 North Road",
+    city: "London",
+    zip_code: "W93 6KU",
+    country_code: "GB",
+    description: "This is a fun little store!"
+  },
+])
```

Comment:
great, thanks for this

Reply:
Yes! We can add more and improve it as we go but this should suffice for now.